front/app/components/Sharing/utils.ts

```diff
@@ -18,14 +18,17 @@ export function getUrlWithUtm(
 ) {
   let resUrl = url;
 
-  resUrl += `?utm_source=${encodeURIComponent(
```

Comment:
These utm params aren't necessary or used at all for the Facebook links anyways, and they are causing issues when trying to share a link via Facebook.

I've checked and these also aren't being used for any sort of tracking purposes.